dozer-types/src/types/field.rs

```diff
@@ -47,7 +48,7 @@ impl Field {
             Field::Decimal(_) => 16,
             Field::Timestamp(_) => 8,
             Field::Date(_) => 10,
-            Field::Bson(b) => b.len(),
+            Field::Json(b) => bincode::serialize(b).unwrap().len(),
```

Comment:
Leaving a comment to remind myself to think of something better

Reply:
ok!